src/components/view/news/news.ts

```diff
@@ -0,0 +1,50 @@
+import './news.css';
+import {Article} from '../../../types/types'
+
+class News {
+    draw(data: Article[] | []) {
+        const news = data.length >= 10 ? data.filter((_item, idx) => idx < 10) : data;
+
+        const fragment = document.createDocumentFragment();
+        const newsItemTemp = document.querySelector('#newsItemTemp') as HTMLTemplateElement;
+
+        if (!newsItemTemp) {
+          return;
+        }
+
+        news.forEach((item, idx) => {
+            const newsClone = newsItemTemp.content.cloneNode(true) as HTMLElement;
+
+            //if (!newsClone) return; why doesn't this check work ???
```

Comment:
We need to check by the like this condition - `newsClone instanceof DocumentFragment`